gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -136,14 +148,26 @@ protected void processMessage(DecodeableKafkaRecord message) {
     // We only expect INSERT and DELETE operations done to this table. INSERTs correspond to any type of
     // {@link DagActionStore.FlowActionType} flow requests that have to be processed. DELETEs require no action.
     try {
+
       if (operation.equals("INSERT")) {
         if (dagActionType.equals(DagActionStore.FlowActionType.RESUME)) {
           log.info("Received insert dag action and about to send resume flow request");
           dagManager.handleResumeFlowRequest(flowGroup, flowName,Long.parseLong(flowExecutionId));
+          //TODO: add a flag for if condition only if multi-active is enabled
+          if(isRefactoredDagManagerEnabled) {
+            ResumeDagTask resumeDagTask = new ResumeDagTask(new DagManager.DagId(flowGroup, flowName, flowExecutionId));
+            dagTaskStream.resumeFlow(resumeDagTask);
+          }
```

Comment:
maybe interpret the flag earlier and turn this into an `Optional` to simplify it with `.ifPresent` (which is part of java, but not guava `Optional`)

Reply:
refactored the code to not use `Optional`